src/app.ts

```diff
@@ -25,6 +25,7 @@ app.use((err: Error, req: Request, res: Response, next: NextFunction) => {
     return next(err);
   }
 
+  console.error(err)
```

Comment:
yes, because at that stage in middleware, only errors that aren't http-errors (dev made errors) should get to. Meaning only unexpected errors are getting logged.